src/LoginServer/Network/LoginPacketHandler.cs

```diff
@@ -266,7 +258,7 @@ public void CB_COMMANDER_CREATE(LoginConnection conn, Packet packet)
 			character.Hair = hair;
 
 			character.MapId = mapData.Id;
-			character.Position = new Position(startingCityData.X, startingCityData.Y, startingCityData.Z);
+			character.Position = new Position(-628, 260, -1025);     // These are the starting coordinates for 'f_siauliai_west'.
```

Comment:
We should definitely make this configurable at some point though, like in Athena^^